Sources/MessagingInApp/Gist/Managers/MessageManager.swift

```diff
@@ -190,6 +196,8 @@ extension MessageManager: EngineWebDelegate {
 
     func routeChanged(newRoute: String) {
         Logger.instance.info(message: "Message route changed to: \(newRoute)")
+
+        willChangeMessage(newTemplateId: newRoute)
```

Comment:
I thought that the name "will change message" and "template id" was a better name for this inline feature. The Gist SDK tends to use "route" when internally on the mobile team, we call that a "template". Also, I thought `willChangeMessage` is more descriptive because when a route is changed, it's because we will be showing a new message.